corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -597,9 +598,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                             value: value,
                         });
 
-                        self.children.findByIndex(i)._setItemset(choices, response.models[i].get('itemsetChoicesKey'));
+                        self._getChildren().findByIndex(i)._setItemset(choices, response.models[i].get('itemsetChoicesKey'));
 
-                        self.children.findByIndex(i)._render();      // re-render with new choice values
+                        self._getChildren().findByIndex(i)._render();      // re-render with new choice values
```

Comment:
nit: I'm sure this will get optimized away but is it possible to use `children` here rather than calling `_getChildren` multiple times?

Reply:
No, because `.children` does not include nested children. But I pulled `self._getChildren().findByIndex(i)` out into a variable.